dist/licenses/LICENSE-github-protobuf.txt

```diff
@@ -0,0 +1,27 @@
+Copyright 2010 The Go Authors.  All rights reserved.
```

Comment:
This file should be removed? As filename seems to be wrong `dist/licenses/LICENSE-github-protobuf.txt`

Reply:
I found there are 2 protobuf libs , and the licenses seem not equal, so add 2 licenses. I am not quite familiar with Proto and I'd like your advice

Reply:
Who are using them? Do we have a chance to remove one later?